core/src/main/java/hudson/tools/DownloadFromUrlInstaller.java

```diff
@@ -148,6 +146,23 @@ public String getId() {
             if(d==null)     return Collections.emptyList();
             return Arrays.asList(((InstallableList)JSONObject.toBean(d,InstallableList.class)).list);
         }
+        
+        /**
+         * The UI uses this information to populate the drop-down.
+         * 
+         * @return JSON array of installable tools, never null
+         * @since 1.617
+         */
+        public JSONArray getInstallablesJson() throws IOException {
```

Comment:
It's new API that only exists for the UI. If someone wants to use it, the annotation can always be removed later, but for now, it's just more to maintain for no reason.